README.md

```diff
@@ -35,35 +41,67 @@ See here for [installation instructions for Docker](https://docs.docker.com/get-
 
 Docker Compose is typically included with Docker Desktop installations, but if you need to install it separately, see [the Docker Compose installation instructions](https://docs.docker.com/compose/install/).
 
-### Ollama
+### (Optional) Ollama for local LLM models
 
-To run models locally, you'll need to have [Ollama](https://ollama.com/) installed on your machine.
+If you want to run LLM models locally, you'll need to have [Ollama](https://ollama.com/) installed on your machine.
 Once you have Ollama installed, you want to run the Ollama server, which will allow the application to access the model.
-If Ollama server is not running as a system service already, you have to start it manually:
+Make sure the Ollama server is listening to all addresses so you can use our Docker images.
+If the Ollama server is running as a system service already, stop it first (in Ubuntu Linux, you'll need to run `sudo systemctl stop ollama.service`).
+Then, start it manually:
 
 ```bash
-ollama serve
+OLLAMA_HOST=0.0.0.0 ollama serve
```

Comment:
Out of curiosity, were you unable to get your local ollama to connect to the app before this? I didn't need this env var, although I imagine it can't hurt to include it.